worlds/kh1/docs/en_Kingdom Hearts.md

```diff
@@ -0,0 +1,88 @@
+
+Other credits include:
+
+Sonicshadowsilver2 for their work finding many memory addresses, working to idenitify and resolve bugs, and coverting the code base to the latest EGS update.
+
+Shananas and the rest of the OpenKH team for providing such an amazing tool for us to utilize on this project.
+
+TopazTK for their work on the `Show Prompt` method and Krujo for their implementation of the method in AP.
+
+JaredWeakStrike for helping clean up my mess of code.
+
+KSX for their interact in battle code.
+
+RavSpect for their title screen image edit.
+
+SunCatMC for their work on ChecksFinder, which I used as a basis for game-to-client communication.
+
+ThePhar for their work on Rogue Legacy AP, which I used as a basis for the apworld creation.
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains most of the options you need to 
+configure and export a config file.
+
+## What does randomization do to this game?
+
+The Kingdom Hearts AP Randomizer randomizes most rewards in the game, and adds several items which are used to unlock worlds, Olympus Coliseum cups, and world progression.
+
+Worlds can only be accessed by finding the corresponding item.  For example, you need to find the `Monstro` item to enter Monstro.
+
+The default goal is to enter End of the World and defeat Final Ansem.
+
+## What items and locations get shuffled?
+
+###Items
+
+Any weapon, accessory, spell, trinity, summon, world, key item, stat up, consumable, or ability can be found in any location.
+
+###Locations
+
+Locations the player can find items include chests, event rewards, Atlantica clams, level up rewards, 101 Dalmation rewards, and postcard rewards.
+
+## Which items can be in another player's world?
+
+Any of the items which can be shuffled may also be placed into another player's world. It is possible to choose to limit
+certain items to your own world.
+## When the player receives an item, what happens?
+
+When the player receives an item, your client will display a message displaying the item you have obtained.  You will also see a notification in the "LEVEL UP" box.
+
+## What do I do if I encounter a bug with the game?
+
+Please reach out to Gicu#7034 on Discord.
+
+## How do I progress a certain world?
+
+###The evidence boxes aren't spawning in Wonderland.
+
+Find `Footprints` in the multiworld.
+
+###I can't enter any cups in Olympus Coliseum.
+
+Firstly, find `Entry Pass` in the multiworld.  Additionally, `Phil Cup`, `Pegasus Cup`, and `Hercules Cup` are all multiworld items.  Finding all 3 grant you access to Hades Cup and the Platinum Match.  Clearing all cups lets you challenge Ice Titan.
+
+###The slides aren't spawning in Deep Jungle.
+
+Find `Slides` in the multiworld.
+
+###I can't progress Atlantica.
+Find `Crystal Trident` in the multiworld.
+
+###I can't progress Halloween Town.
```

Comment:
```suggestion
###I can't progress in Halloween Town.
```

Reply:
Fixed in a separate commit to apply the same grammar for the line above (I can't progress in Atlantica)